examples/extend-express-app/routes/tasks.ts

```diff
@@ -11,10 +11,7 @@ import type { Context } from '.keystone/types';
   We're also demonstrating how you can query related data through the schema.
 */
 
-export async function getTasks(req: Request, res: Response) {
-  // This was added by the context middleware in ../keystone.ts
-  const { context } = req as typeof req & { context: Context };
```

Comment:
We don't want to encourage `as` in user code, the preferred approach is to use a `Context` parameter directly